packages/vc-verification/src/models/credentials-types-ti.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * This module was automatically generated by `ts-interface-builder`
```

Comment:
@JGiter Could you elaborate on how this automatic generate was performed? Did you need to manually trigger the generation? How will this file be updated in the future?

Reply:
This file was generated by ts-interface-builder. I din't include generation in build script, but it is possibly to automate re-generation

Reply:
Ok, we can choose appropriate interface checker in dedicated PR. I'll revert to initial fix